code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -17,6 +17,7 @@
 	var/infinite = FALSE
 	/// If TRUE, won't play a noise when injecting.
 	var/stealthy = FALSE
+	var/used_up = FALSE
```

Comment:
Auto doc

Reply:
Not too sure what that means, if I need to do something for it.

Reply:
It means you have to explain what this variable does.

See the  variable above "stealthy". It has this above it
/// if TRUE, won't play a noise when injecting

In the same way you have to write above your variable with /// what it does

Reply:
Oh ok.